src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -1322,39 +1364,13 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 					{
 						h.append( m_stepSize );
 						h.append( m_volumePadding );
-
-						if( m_volumeGrids )
-						{
-							m_volumeGrids->hash( h );
-						}
-
-						if( m_velocityGrids )
-						{
-							m_velocityGrids->hash( h );
-						}
-
-						h.append( m_velocityScale );
-						h.append( m_velocityFPS );
-						h.append( m_velocityOutlierThreshold );
```

Comment:
Do we not need an `m_volume.hash( h )` call here?

Reply:
I've addressed that in an extra commit -> a126a40. The actual issue was that I've missed applying those attributes to anything but @donboie's shiny new VDBObjects which I was too focused on, I guess. 
I'm not merging in this one just yet so you can have a look at it. 

Also, when I was playing with the ArnoldVDB node it got me thinking that with this PR we're breaking all Objects that come with those parameters already set, aren't we? We're basically resetting those parameters to the defaults if the respective attribute isn't set to a non-default value?
In the case of ArnoldVDB we could potentially switch that node to set the velocity related parameters as attributes instead, but maybe we need a more general solution in the backend?

Reply:
> Also, when I was playing with the ArnoldVDB node it got me thinking that with this PR we're breaking all Objects that come with those parameters already set, aren't we? We're basically resetting those parameters to the defaults if the respective attribute isn't set to a non-default value?

Hmm. True. While we are planning to phase out ArnoldVDB in favour of Don's shiny VDBObjects, I don't think we should break any existing workflows just yet. How about storing the new attributes as `optional<Whatever>` and then only setting the Arnold parameters if the attribute itself was actually set in Gaffer?

Reply:
I've given that a try in 76ae41a. Not sure if you like the additional templated function, but it seemed to me like this is the cleanest way of doing it?

Also, I'd be curious to hear your opinion on 

`velocityScale.get() > 0`

versus

`velocityScale > 0.0f`

While the latter is fancy, the former is more explicit and maybe easier to understand? Maybe doesn't matter - I opted for the former.